app/views/index.scala.html

```diff
@@ -154,10 +158,14 @@
             </a>
         </div>
         <script language="javascript">
-            if (@("%.0f".format(StreetEdgePriorityTable.streetDistanceCompletionRateUsingPriority * 100)) < 100) {
-                document.getElementById('conditional-text').innerHTML = "@Messages("landing.stats.content.unfinished", "%,.0f".format(auditedDistance), cityName, stateAbbreviation, "%.1f".format(StreetEdgePriorityTable.streetDistanceCompletionRateUsingPriority * 100))";
-            } else {
-                document.getElementById('conditional-text').innerHTML = "@Messages("landing.stats.content.finished", "%,.0f".format(auditedDistance), cityName, stateAbbreviation)";
+	    if (("@(Play.configuration.getString("city-params.open-status." + cityId).get)" === "partially")&&(@("%.0f".format(StreetEdgePriorityTable.streetDistanceCompletionRateUsingPriority * 100)) >= 100)) {
```

Comment:
You have a lot of extra parentheses throughout this file. You added parens around each condition in the `if`, which isn't necessary, and you also added parens around your call to `Play.configuration` which isn't necessary.

With template Scala, you only need parens directly after the `@` symbol if it's more than just a call to a single function (like when we use `@("%.0f".format( ... ))`. But just calling `@Play.configuration.getString(...)` works fine.

So this line
```
if (("@(Play.configuration.getString(...).get)" === "partially")&&(@("%.0f".format(...)) >= 100)) {
```

Should probably look more like this
```
if ("@Play.configuration.getString(...).get" === "partially" && @("%.0f".format(...)) >= 100) {
```